src/static/about.html

```diff
@@ -0,0 +1,177 @@
+      <h3>InVEST Version: </h3> 
+    </span>
+    <br/><br/>
+    <span id='workbench-version'>
+      <h3>Workbench Version: </h3> 
+    </span>
+    <br/> <br/>
+
+    Documentation: 
+    <a
+      href="http://releases.naturalcapitalproject.org/invest-userguide/latest/"
+      onClick="handleClick(event)">
+      http://releases.naturalcapitalproject.org/invest-userguide/latest/
+    </a>
+    <br/>
+    Homepage: 
+    <a
+      href="https://naturalcapitalproject.stanford.edu/"
+      onClick="handleClick(event)">
+      https://naturalcapitalproject.stanford.edu/
+    </a>
+    <br/>
+    Copyright 2021, The Natural Capital Project
+    <br/>
+    License: 
+    <a
+      href="https://github.com/natcap/invest/blob/master/LICENSE.txt"
+      onClick="handleClick(event)">
+      BSD 3-clause
+    </a>
+    <br/>
+    Project pages: 
+    <br/>
+    <a
+      href="https://github.com/natcap/invest"
+      onClick="handleClick(event)">
+      github.com/natcap/invest
+    </a>
+    <br/>
+    <a
+      href="https://github.com/natcap/invest-workbench"
+      onClick="handleClick(event)">
+      github.com/natcap/invest-workbench
+    </a>
+    <br/>
+
+    <h2>Open-Source Licenses</h2>
+    <table>
+        <tr>
+          <td>PyInstaller</td>
+          <td>GPL</td>
+          <td>
+            <a
+              href='http://pyinstaller.org'
+              onClick="handleClick(event)">
+              http://pyinstaller.org
+            </a>
+          </td>
+        </tr>
+        <tr>
+          <td>GDAL</td>
+          <td>MIT and others</td>
+          <td>
+            <a
+              href='http://gdal.org'
+              onClick="handleClick(event)">
+              http://gdal.org
+            </a>
+          </td>
+        </tr>
+        <tr>
+          <td>numpy</td>
+          <td>BSD</td>
+          <td>
+            <a
+              href='http://numpy.org'
+              onClick="handleClick(event)">
+              http://numpy.org
+            </a>
+          </td>
+        </tr>
+        <tr>
+          <td>pyamg</td>
+          <td>BSD</td>
+          <td>
+            <a
+              href='http://github.com/pyamg/pyamg'
+              onClick="handleClick(event)">
+              http://github.com/pyamg/pyamg
+            </a>
+          </td>
+        </tr>
+        <tr>
+          <td>pygeoprocessing</td>
+          <td>BSD</td>
+          <td>
+            <a
+              href='https://github.com/natcap/pygeoprocessing'
+              onClick="handleClick(event)">
+              https://github.com/natcap/pygeoprocessing
+            </a>
+          </td>
+       </tr>
+        <tr>
+          <td>PyQt</td>
+          <td>GPL</td>
+          <td>
+            <a
+              href='https://riverbankcomputing.com/software/pyqt/intro'
+              onClick="handleClick(event)">
+              https://riverbankcomputing.com/software/pyqt/intro
+            </a>
+          </td>
+       </tr>
```

Comment:
Hm I suppose this is out of date as well, since we  now use PySide2.  Uffda ... we should probably look back into licensing documentation issues at some point.

Reply:
Come to think of it, maybe we can remove this one altogether since we'll ultimately ship the workbench without PyQt or PySide2.